tests/vm/CMakeLists.txt

```diff
@@ -4,3 +4,19 @@ add_executable(aseba-test-natives-count
 )
 target_link_libraries(aseba-test-natives-count asebacompiler asebavm asebavmdummycallbacks ${ASEBA_CORE_LIBRARIES})
 add_test(natives-count ${EXECUTABLE_OUTPUT_PATH}/aseba-test-natives-count)
+
+# test the deque native functions
+add_test(NAME deque-empty COMMAND $<TARGET_FILE:asebatest> --memcmp
```

Comment:
No, not necessary, CMake can find the target `asebatest`. However there is a problem with the `simulate.py` tests in `tests/compiler`, CMake isn't generating the correct value of `${CMAKE_CURRENT_BINARY_DIR}` for Xcode. I was trying a global fix a generator time but it didn't work